customer.py

```diff
@@ -0,0 +1,21 @@
+from person import Person
+from vehicle import Vehicle
+
+class Customer(Person):
+    def __init__(self, name, email, vehicle:Vehicle, loyalty_points, membership_date, owner, make, model, color, registration_num):
```

Comment:
This is a type annotation. It is incorrect in this context ctor should not have extra fields.